### PR TITLE
Added, NM Chonchon, Ankou, Hyakinthos, and fixed Lamina

### PR DESCRIPTION
**Pull request by Fiocitrine**
I think this is it. I added three NMs and fixed Lamina to spawn and drop the ring.

### CODE REVIEW

File: sql/mob_groups.sql [hunk 1]
```diff
@@ -12299,6 +12299,14 @@ INSERT INTO `mob_groups` VALUES (14521,5722,288,0,128,0,0,0,105,105,0);
 INSERT INTO `mob_groups` VALUES (14522,5723,288,0,128,0,0,99999,103,104,0);
 INSERT INTO `mob_groups` VALUES (14523,5724,288,0,128,0,0,0,103,104,0);
 
+-- Ankou
+INSERT INTO `mob_groups` VALUES (14524,5726,190,3600,0,4560,650,650,25,25,0);
```
Comment (Hozu) [inline sql/mob_groups.sql]: I think you have the wrong mob droplist here - 4560 is one of the Dynamis - Qufim NMs.

File: sql/mob_groups.sql [hunk 2]
```diff
@@ -12299,6 +12299,14 @@ INSERT INTO `mob_groups` VALUES (14521,5722,288,0,128,0,0,0,105,105,0);
...
+
+-- Hyakinthos
+INSERT INTO `mob_groups` VALUES (14525,5725,98,7200,0,4562,6200,0,58,59,0);
```
Comment (Hozu) [inline sql/mob_groups.sql]: Same here.

**Comment by Fiocitrine** [comment]
Actually, my DropIDs in mob_groups didn't match the DropIDs I had in mob_droplist. thanks hozu.

**Comment by bendangelo** [comment]
Looks good, thanks

**Pull request merged**